pom.xml

```diff
@@ -75,7 +75,7 @@
         <java.version>8</java.version>
         <project.build.resourceEncoding>UTF-8</project.build.resourceEncoding>
         <aether.version>0.9.0.M2</aether.version>
-        <apache.curator.version>5.5.0</apache.curator.version>
+        <apache.curator.version>5.3.0</apache.curator.version>
```

Comment:
There could be a comment here about why we're in this version, so the rationale won't get forgotten. (Link to the Curator JIRA is best)

Reply:
@gianm Done. FYI, https://github.com/apache/druid/pull/16444